src/components/shared/ScrollTarget.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+	<div class="scroll-target">
+		<div :id="targetId"/>
+	</div>
```

Comment:
After fiddling with this and `window.scroll/element.scroll` I think using `window.scrollIntoView` provides the best compatibility as it works with overflow elements as well as the window. Safari doesn't smooth scroll the modal, but it jumps instead.

If we use `window.scroll` we would need to calculate the focus element's distance from the top of the document, then add the offset and it'll scroll fine. But in the modals with the overflow element we'd need to reference it somehow in both the modal and the error summary so the summary can tell the modal where to scroll to, the modal would then also need to do position calculations to get the correct point.

I think `scrollIntoView` matches closer to the ideal of using what the browser gives us for free rather than implementing our own solution.